package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "https://github.com/edgedb/edgedb-js.git"
   },
-  "version": "0.7.3",
+  "version": "0.7.4",
```

Comment:
Hm, why this change?

Reply:
I thought we would publish a new version of the package to npm, but I made a mistake not asking. Besides, I see there is already 0.7.4 in npmjs.com. I revert to the previous value.

Reply:
That's ok! I think `yarn publish` increments the version itself, so basically we never need to change it manually in package.json.